examples/tests/invalid/subdir2/file.w

```diff
@@ -0,0 +1,7 @@
+class MyPrivateClass {}
+
+struct MyPrivateStruct {}
```

Comment:
Trying to understand: does this change also allow two private symbols (classes, namespaces, etc) with the same name to coexist in the same subdirectory?

Reply:
Turns out, yes.

Furthermore, if you have a private `Foo` in one file and a public `Foo` in a sibling file, it's also OK. (just added a test for this)